src/pages/AdminManageInstitutions.tsx

```diff
@@ -0,0 +1,97 @@
+import React, {useEffect, useState} from 'react';
+import {Institution as InstitutionAPI} from 'src/libs/ajax/Institution';
+import {Institution} from 'src/types/model';
+import {Styles} from '../libs/theme';
+import {getSearchFilterFunctions, Notifications} from 'src/libs/utils';
+import manageInstitutionsIcon from 'src/images/icon_manage_dac.png';
+import SearchBar from 'src/components/SearchBar';
+import InstitutionTable from 'src/components/institution_table/InstitutionTable';
+import {tableHeaderTemplate, tableRowLoadingTemplate} from 'src/components/institution_table/InstitutionTableUtils';
+import DarTableSkeletonLoader from 'src/components/TableSkeletonLoader';
+import {Link} from 'react-router-dom';
+import {extractError} from 'src/utils/ErrorUtils';
+
+export default function AdminManageInstitutions() {
+  const [institutionList, setInstitutionList] = useState<Institution[]>([]);
+  const [filteredList, setFilteredList] = useState<Institution[]>([]);
+  const [tableSize, setTableSize] = useState<number>(10);
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [searchTerm, setSearchTerm] = useState<string>('');
+
+  useEffect(() => {
+    const init = async () => {
+      try {
+        setIsLoading(true);
+        const listOfInstitutions = await InstitutionAPI.list();
+        setInstitutionList(listOfInstitutions);
+        setFilteredList(filter(listOfInstitutions, searchTerm));
+        setIsLoading(false);
+      } catch (error) {
+        const message = extractError(error);
+        Notifications.showError({text: 'Error: Unable to retrieve institutions from server: ' + message});      } finally {
+        setIsLoading(false);
+      }
+    };
+    init();
+  }, []);
+
+  const handleSearchChange = (query: { current: { value: string } }) => {
+    const value = query.current.value;
+    setSearchTerm(value);
+    setFilteredList(filter(institutionList, searchTerm));
+  };
+
+  const filter = (list: Institution[], value: string): Institution[] => {
+    if (value) {
+      return getSearchFilterFunctions().institutions(value, list);
+    }
+    return list;
+  };
```

Comment:
Previously, we were using the institution json to use for matching. That would allow for users to filter on the property names such as `createDate`, `signingOfficials`, etc, which will match on every institution with those fields populated.